src/Client/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace InterviewApp.Client.Extensions;
+
+public static class StringExtensions
+{
+    public static bool IsNullOrEmpty(this string value)
```

Comment:
Make it nullable string
```suggestion
    public static bool IsNullOrEmpty(this string? value)
```